src/KernelAbstractions.jl

```diff
@@ -535,6 +535,14 @@ function priority!(::Backend, prio::Symbol)
     return nothing
 end
 
+
+"""
+    ongpu()
+
+Returns if this kernel is currently executing on a GPU.
+"""
+function ongpu end
```

Comment:
Prefix with `is` to make it easier to tab-complete?
```suggestion
function isongpu end
```